src/samples/public-client/IntegratedWindowsAuthFlow.java

```diff
@@ -40,16 +40,17 @@ public static void main(String args[]) throws Exception {
 
     private static AuthenticationResult getAccessTokenByIntegratedAuth() throws Exception {
         PublicClientApplication app = PublicClientApplication.builder(TestData.PUBLIC_CLIENT_ID)
-                .authority(TestData.AUTHORITY_COMMON)
+                .authority(TestData.AUTHORITY_ORGANIZATION)
+                .telemetryConsumer(new Telemetry.MyTelemetryConsumer().telemetryConsumer)
                 .build();
 
-        Future<AuthenticationResult> future = app.acquireToken
-                (IntegratedWindowsAuthenticationParameters.builder
-                        (Collections.singleton(TestData.GRAPH_DEFAULT_SCOPE), TestData.USER_NAME)
-                        .build());
+        IntegratedWindowsAuthenticationParameters parameters =
+                IntegratedWindowsAuthenticationParameters.builder(
+                        Collections.singleton(TestData.GRAPH_DEFAULT_SCOPE), TestData.USER_NAME)
+                        .build();
 
-        AuthenticationResult result = future.get();
+        Future<AuthenticationResult> future = app.acquireToken(parameters);
 
-        return result;
+        return future.get();
```

Comment:
>future.get() [](start = 15, length = 12)

is this good for debugging? Wouldn't you prefer to have the result in the variable instead?

Reply:
You're right, changing it back.